tools/base/envoy_python.bzl

```diff
@@ -71,3 +72,28 @@ def envoy_py_binary(
 
     if test:
         envoy_py_test(name, package, visibility, envoy_prefix = envoy_prefix)
+
+def envoy_entry_point(
+        name,
+        pkg,
+        main = "//tools/base:entry_point.py",
+        entry_point = base_entry_point,
+        script = None,
+        data = None,
+        args = None):
+    script = script or pkg
+    entry_point_name = "%s_entry_point" % name
+    native.alias(
+        name = entry_point_name,
+        actual = entry_point(
+            pkg = pkg,
```

Comment:
Can you add some macro level documentation on the parameters? Like, why is there both an entry point name and a pkg, and why does the user of this macro care?

Reply:
sure, ill add a docstring

Reply:
done, ill also fixed the proc handling in the dynamic loader script